src/chaplin/lib/route.coffee

```diff
@@ -7,9 +7,7 @@ Controller = require 'chaplin/controllers/controller'
 utils = require 'chaplin/lib/utils'
 
 isEmpty = (object) ->
-  for own key, value of object
-    return false
-  true
+  not Object.keys(object).length
```

Comment:
Let's revert all ES5 additions. Chaplin is used in many legacy apps at this point.